client/blocks/conversations/empty.jsx

```diff
@@ -37,7 +37,7 @@ class ConversationsEmptyContent extends Component {
 				title={ this.props.translate( 'Welcome to Conversations' ) }
 				line={ this.props.translate(
 					"When WordPress posts spark lively conversations, they'll appear here. " +
-						'To get started, like or comment on some posts.'
+						'To get started, Follow or comment on some posts.'
```

Comment:
Is there a reason to make "follow" a proper noun?

Reply:
I was thinking because it's the name of a feature, but that's probably debatable. I don't feel strongly about it. I'm happy to make it lowercase if that makes more sense.

Reply:
I updated this to lowercase 👍 https://github.com/Automattic/wp-calypso/pull/97353/commits/78509c666e8d7923ed20dc8e3d27d30ce4c76552